README.rst

```diff
@@ -209,8 +209,9 @@ Check HSTS list
 DNS lookup
 ----------
 
-* Browser checks if the domain is in its cache. (to see the DNS Cache in
-  Chrome, go to `chrome://net-internals/#dns <chrome://net-internals/#dns>`_).
+* Browser checks if the domain is in its cache. (To see the DNS Cache in
+  Chrome, go to `chrome://net-internals/#dns <chrome://net-internals/#dns>`_.
+  To see it in Firefox, go to `about:networking#dns <about:networking#dns>`_).
```

Comment:
Maybe it is useful to add Edge (edge://net-internals/#dns) and Brave (brave://net-internals/#dns) too, to see the presence of Chromium in all major browsers.